bluesky/tests/test_metadata.py

```diff
@@ -0,0 +1,14 @@
+import bluesky
+import ophyd
+from bluesky import RunEngine
+
+def test_blueskyversion():
```

Comment:
Please use the RunEngine pytest fixture.

```python
def test_blueskyversion(RE):
    ...
```

Reply:
updated